apps/dashboard/src/components/conditions-editor/combinator-selector.tsx

```diff
@@ -0,0 +1,21 @@
+import { type CombinatorSelectorProps } from 'react-querybuilder';
+
+import { RepeatRoundLine } from '@/components/icons/repeat-round-line';
+import { Select, SelectContent, SelectTrigger, SelectValue } from '@/components/primitives/select';
+import { cn } from '@/utils/ui';
+import { toSelectOptions } from '@/components/conditions-editor/select-option-utils';
+
+export const CombinatorSelector = ({ disabled, value, options, handleOnChange }: CombinatorSelectorProps) => {
```

Comment:
This represents the `Or/And` combinator dropdown.
![Screenshot 2025-01-14 at 12 20 00](https://github.com/user-attachments/assets/4463498d-90d2-46fd-bc33-e0f2a6eb2768)